setup.py

```
@@ -1,60 +1,62 @@
 
 # Read the requirements from the requirements.txt file
 with open("requirements.txt") as fp:
-    base_requirements = fp.read().splitlines()
+    install_requires = fp.read().splitlines()
 
 # Platform-specific dependencies
 if platform.system() == "Windows":
-    base_requirements.append("AppOpener==1.7")
-
+    install_requires.append("AppOpener==1.7")
 elif platform.system() == "Darwin":  # Darwin is the system name for macOS
```

Ruff (0.8.2)

Undefined name `platform`

(F821)